### ISSUE TITLE
How to configure `sails lift` to use iojs instead of node?

### ISSUE THREAD

**Issue by miguelmota** [balderdashy/sails#2978]
I'm using `jsdom` library which requires `iojs`. I can do `iojs app.js` which works just fine but I want to do `sails lift` instead but it throws jsdom error that says I need iojs.

**Comment by tjwebb** [comment]
You need to alias `node` to `iojs`. Version managers such as https://github.com/tj/n will do this for you.

Another option is to use a 3.x version of jsdom before they decided to break compatibility with node.

**Comment by sailsbot** [comment]
Thanks for posting, @miguelmota.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
